settings/_settings.global.scss

```diff
@@ -2,17 +2,23 @@
 //   #GLOBAL
 //   ======================================================================== */
 
-// Project-level settings and setup.
+// The global settings file contains any project-wide variables.
+
 
 // Base typographical styles and baseline grid. You need to define these values
 // in pixels: inuitcss will convert them to more appropriate units.
```

Comment:
Should we describe what inuitcss converts them to? I think people want to know if the results are `rem`/`unitless`, `px` or `em` or anything else.